app/database_etl/airtable_records_handler/airtable_records_getter.py

```diff
@@ -84,3 +91,47 @@ def get_all_records():
         }
         send_api_error_slack_notif(body, data, error=e, request_info=request_info, channel='#dev-logging-etl')
         return request_info
+
+def ingest_sample_frame_goi_filter_options():
+    url = "https://api.airtable.com/v0/appFfYIupTrVQ0HJx/Sample%20Frame%20GOI?fields=Order&fields=Name&sort%5B0%5D%5Bfield%5D=Order&fields=French%20Name"
+    headers = {'Authorization': 'Bearer {}'.format(AIRTABLE_API_KEY)}
+
+    # Make request and retrieve records in json format
+    r = requests.get(url, headers=headers)
+    data = r.json()
+
+    # Try to get records from data if the request was successful
+    try:
```

Comment:
This request/retry logic seems pretty similar to the main Airtable request function (and I feel like we may use it more in the future) - let's consider making this a more general function

Reply:
split it into 2 functions: `_airtable_get_request` and `handle_airtable_request`